spec/spec_helper.rb

```diff
@@ -1,4 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+
 require 'xcake'
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.